x-pack/plugins/cases/server/routes/api/cases/comments/delete_all_comments.ts

```diff
@@ -18,19 +18,25 @@ export function initDeleteAllCommentsApi({
   router,
   userActionService,
   logger,
+  subCasesEnabled,
 }: RouteDeps) {
+  const querySchema = subCasesEnabled
```

Comment:
Do we document this API? If yes, then conditional params will be difficult to documented it. Why don't we leave the schema as it is but do not take into account the subCaseId at all.

Reply:
I was thinking the same way, can we just have a global const for now